server/core/src/main/scala/org.apache.daffodil.debugger.dap/Parse.scala

```diff
@@ -157,31 +157,45 @@ object Parse {
   object Debugee {
 
     case class LaunchArgs(
-        schemaPath: Path,
-        dataPath: Path,
+        defaultSchemaPath: Path,
+        defaultDataPath: Path,
         stopOnEntry: Boolean,
         infosetOutput: LaunchArgs.InfosetOutput,
-        tdmlConfig: LaunchArgs.TDMLConfig
+        tdmlConfig: Option[LaunchArgs.TDMLConfig]
     ) extends Arguments {
+      def dataPath: Path =
```

Comment:
Instead of making the consumer of the data type check the `TDMLConfig` or not, we push that into the `LaunchArgs`.